### PR TITLE
Update deepl from 3.1.133440 to 3.2.157164

### PR DESCRIPTION
**Pull request by ondrejfuhrer**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/deepl.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask "deepl" do
-  version "3.1.133440"
+  version "3.2.157164"
```
Comment (neutric) [inline Casks/deepl.rb:2]: I also get `3.1.133440` locally. How'd you come up with the new version?

Comment (ondrejfuhrer) [reply Casks/deepl.rb:2]: hey @neutric , through `brew livecheck deepl`

Comment (ondrejfuhrer) [reply Casks/deepl.rb:2]: Still reporting that version btw:
```
> brew livecheck deepl
deepl : 3.1.133440 ==> 3.2.157164
```

Comment (neutric) [reply Casks/deepl.rb:2]:
```
%  brew bump deepl
==> deepl
Current cask version   :  3.1.133440
Latest livecheck version: 3.2.157164
Latest Repology version:  3.3.4484
```
Now I get that one, too. Didn't when I asked a couple of hours ago.

**Pull request merged**